packages/web3-eth-accounts/package.json

```diff
@@ -60,7 +60,7 @@
 	"dependencies": {
 		"@ethereumjs/rlp": "^4.0.1",
 		"crc-32": "^1.2.2",
-		"ethereum-cryptography": "^1.1.2",
+		"ethereum-cryptography": "^2.0.0",
```

Comment:
with eth-cryptography 2.0 : node.js 14 and older support was dropped, we will need to mention in changelog. 
node 14's maintenance is already ending 2023-04-30 so we are good for that.

Reply:
Ok I'm running it or trying to now